### ISSUE TITLE
Ability to cancel import to the table storage

### ISSUE THREAD

**Issue by koljada** [Microsoft/AzureStorageExplorer#733]
**Storage Explorer Version**: 1.4.2
**Platform/OS Version**: Win10
**Architecture**: ia32
**Regression From**: 

In my opinion it would be useful to have an ability to cancel import(probably export as well) to the table storage. Consider a situation when the user imports a lot of data and suddenly realizes that there are some incorrect values.  At this point he cannot just stop the export: he has to wait until the operation complete and only then delete all inserted data.

**Comment by jhoak** [comment]
+1. There should at least be a warning about large tables or if there's >1k rows to export. I just started an export, with no confirmation, and didnt know that I cannot cancel it. Watching my entity count go up (currently 2.7 million) while I wait for my PC or database to explode.

**Comment by nielsvdc** [comment]
Yes, adding a cancel option for import and export would be very nice. Then you don't have to reload Storage Explorer to stop this kind of process.
